### ISSUE TITLE
macos画面卡顿

### ISSUE THREAD

**Issue by dreamfly-jiefa** [rustdesk/rustdesk#10047]
### Bug Description

1、自建了服务器
2、同样的网络环境
3、使用安卓平板（控制端）远程控制windows（被控端），非常的流畅
4、使用安卓平板或者windows（控制端）远程控制macos（被控端），画面就会非常的卡顿，帧率只有10帧左右
存在的问题的被控端是MacBook Air M1，系统是Macos 12.7

### How to Reproduce

为什么rustdesk控制windows很流畅，控制Macos就非常的卡顿？

### Expected Behavior

希望控制Macos也可以非常的流畅

### Operating system(s) on local (controlling) side and remote (controlled) side

Windows 10->MacOS 12.7

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

Any->1.3.2

### Screenshots

为什么rustdesk控制windows很流畅，控制Macos就非常的卡顿？

**Comment by 21pages** [comment]
What's the resolution and codec of Mac?

**Comment by dreamfly-jiefa** [comment]
> What's the resolution and codec of Mac?
我尝试过2560x1600或者1920x1200的分辨率，编码是H265，在macos中一样会出现湖面卡顿的现象，但是在windows中就很流畅。
Sorry,my English may be not good.
I have tried resolutions of 2560x1600 or 1920x1200 with H265 encoding, which can cause stuttering on the surface of the lake in macOS, but it runs smoothly in Windows.

**Comment by dreamfly-jiefa** [comment]
> What's the resolution and codec of Mac?

To be precise, I have tried any resolution and encoding on macOS, but it always results in stuttering.

**Comment by 21pages** [comment]
Are the blue icons the same when you control windows and macos?

![image](https://github.com/user-attachments/assets/f31f10c2-1e33-432b-84b1-b0e4f2ee0f1f)

**Comment by dreamfly-jiefa** [comment]
> Are the green icons the same when you control windows and macos?
> 
> ![image](https://private-user-images.githubusercontent.com/14891774/389434800-f31f10c2-1e33-432b-84b1-b0e4f2ee0f1f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI1MjI4NDAsIm5iZiI6MTczMjUyMjU0MCwicGF0aCI6Ii8xNDg5MTc3NC8zODk0MzQ4MDAtZjMxZjEwYzItMWUzMy00MzJiLTg0YjEtYjBlNGYyZWUwZjFmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTI1VDA4MTU0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU5N2Q5NTUyMTYwNTI3MjUyY2RmMzVjYzZlMjA1OTNlZGQ0YmY4ZWRjYWRmZGYwMTJhZjg3NjRlOWU3MzA1ZGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.eHY2oaoEVQAjP4Bh5-1TOKEitApOcxli6XQvheN8iZM)
> 
> ![image](https://private-user-images.githubusercontent.com/14891774/389435046-4c02c2ce-b9c5-4aba-865f-623e51e9708c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI1MjI4NDAsIm5iZiI6MTczMjUyMjU0MCwicGF0aCI6Ii8xNDg5MTc3NC8zODk0MzUwNDYtNGMwMmMyY2UtYjljNS00YWJhLTg2NWYtNjIzZTUxZTk3MDhjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTI1VDA4MTU0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE3ZTc0MzQyMGU2NWNlNzJhOTMwM2Y1OTFhM2FlYzc0MTdhNDc0NGZhY2QwMjkwZGM2ZDlhOTRlZDc1NzliMTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.qyuhL4JL9rCCfWZ7tKBw57EWKSot_RGODi_9Z-K_Mlk)

Yes,the green icons are same。All of them are check marks（First picture）

**Comment by 21pages** [comment]
Could you upload a video?
